### ISSUE TITLE
Add view command list

### ISSUE THREAD

**Issue by risfy** [stormstricker/JavaStudyBuddies#3]
Add a command like (`!w help`) or (`!w h`) or (`!w commands`) to give user list of commands available and how to use bot

**Comment by epicmanmoo** [comment]
Yep, that is a necessary feature and once we have all the commands to make, we can go ahead and start implementing that as an embed.
